packages/terra-functional-testing/src/reporters/spec-reporter/get-output-dir.js

```diff
@@ -0,0 +1,18 @@
+const fs = require('fs');
+const path = require('path');
+
+/**
+ * Determines the output directory for reporter results.
+ * @returns {string} - An output directory.
+ */
+const getOutputDir = () => {
+  const cwd = process.cwd();
+
+  if (fs.existsSync(path.join(cwd, 'test'))) {
```

Comment:
are we for sure keeping the tests/test thing? IMO i'd default. one and let people config if absolutely necessary

Reply:
Removed dynamic test directory logic. The function returns `tests` now. A user can configure this using the `reporterOptions` in the wdio config.

https://github.com/cerner/terra-toolkit/pull/535/commits/88f0eac64b0f4c17fc2d2da031653fd0be66b18c